src/Akeneo/Bundle/ElasticsearchBundle/IndexConfiguration/IndexConfiguration.php

```diff
@@ -33,6 +33,24 @@ public function __construct(array $settings, array $mappings, array $aliases)
     }
 
     /**
+     * Get the full and compiled index configuration of Settings, Mappings and Aliases.
```

Comment:
can you add an @see https://www.elastic.co/guide/en/elasticsearch/client/php-api/5.0/php_json_objects.html#_arrays_of_empty_objects ?